source/CIPlanetNode.cpp

```diff
@@ -18,13 +18,18 @@
 
 void PlanetNode::draw(const std::shared_ptr<cugl::SpriteBatch>& batch,
                       const cugl::Mat4& transform, cugl::Color4 tint) {
-  PolygonNode::draw(batch,transform, tint);
+  AnimationNode::draw(batch,transform, tint);
 }
 
 void PlanetNode::update(float timestep) {
   _timeElapsed += timestep;
   if (_timeElapsed > SPF) {
     _timeElapsed = 0;
+    
+    unsigned int coreFrame = getFrame();
+    coreFrame = (coreFrame == CORE_END) ? CORE_START: coreFrame += 1;
```

Comment:
NIT: coreFrame + 1 instead of coreFrame += 1. Also what henry said with the : spacing